revolv/templates/base/base.html

```diff
@@ -82,16 +82,6 @@
 var options={ "publisher": "{% sharethis_pub_id  %} ", "position": "left", "ad": { "visible": false, "openDelay": 5, "closeDelay": 0}, "chicklets": { "items": ["", "sharethis"]}};
 var st_hover_widget = new sharethis.widgets.hoverbuttons(options);
 </script>
-<script>
```

Comment:
Why did you remove this? It's important for tracking pageviews via Google Analytics.